src/components/Cart/Cart.jsx

```diff
@@ -4,93 +4,99 @@ import Button from '@material-ui/core/Button';
 import SwipeableDrawer from '@material-ui/core/SwipeableDrawer';
 import List from '@material-ui/core/List';
 import Divider from '@material-ui/core/Divider';
-import ListItem from '@material-ui/core/ListItem';
-import ListItemText from '@material-ui/core/ListItemText';
-import ListItemAvatar from '@material-ui/core/ListItemAvatar';
-import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction';
-import IconButton from '@material-ui/core/IconButton';
-import Avatar from '@material-ui/core/Avatar';
 import Typography from '@material-ui/core/Typography';
-import { useQuery } from '@apollo/react-hooks';
-import DeleteIcon from '@material-ui/icons/Delete';
-import { CART_OPEN_QUERY, GET_PRODUCTS_BY_ID, CART_ITEM } from './GraphQL';
+import { useQuery, useMutation } from '@apollo/react-hooks';
+import { CART_OPEN_QUERY, GET_CHECKOUT_ITEMS } from './GraphQL';
 import useStyles from './Styles';
+import Spinner from '../Spinner';
+import LineItem from '../LineItem';
+import { CHECKOUT_LINE_ITEMS_REPLACE, GET_CHECKOUT_ID } from '../Shop/GraphQl';
 
 export default function Cart() {
   const classes = useStyles();
-  const { data: { cartItems } = 0 } = useQuery(CART_ITEM);
-  const productIds = store.get('ids');
+  const cartId = store.get('checkoutId');
+  const { data: { isOpenCart } = false, client } = useQuery(CART_OPEN_QUERY);
+  const {
+    data: { checkoutId }
+  } = useQuery(GET_CHECKOUT_ID);
+  const { data, loading } = useQuery(GET_CHECKOUT_ITEMS, {
+    variables: { id: cartId }
+  });
+
+  const [
+    checkoutLineItemsReplace,
+    { loading: checkoutReplaceLoad }
+  ] = useMutation(CHECKOUT_LINE_ITEMS_REPLACE, {
+    onCompleted: () => {
+      client.writeData({ data: { bucketItemsCount: 0 } });
+    },
+    refetchQueries: [
+      {
+        query: GET_CHECKOUT_ITEMS,
+        variables: { id: checkoutId }
+      }
+    ]
+  });
 
-  const toggleDrawer = (client, open) => event => {
-    if (event && event.currentTarget.className === 'makeStyles-list-160') {
+  const toggleDrawer = open => event => {
+    if (event && event.currentTarget.className === 'makeStyles-list-164') {
```

Comment:
Give it a name because it's not very obvious what it does. Extract to variable or to func